web-components/src/components/taskitem/TaskItem.ts

```diff
@@ -31,6 +30,37 @@ export namespace TaskItem {
     @property({ type: String }) iconSrc = "";
     @property({ type: String }) tabIndexForContainer = "0";
 
+    /**
+    * @deprecated Use `itemTitle` instead.
+    * Using title will may cause an unwanted tooltip.
+    */
+    @property({ type: String, reflect: true })
+    get title(): string {
+      return this.displayTitle;
+    }
+    set title(title: string) {
+      this.updateTitle(title);
+    }
+    /**
+    * The visible text for the Title.
+    */
+    @property({ type: String, reflect: true})
+    get itemTitle() {
+      return this.displayTitle;
+    }
+    set itemTitle(itemTitle: string) {
+      this.updateTitle(itemTitle);
+    }
+
+    private updateTitle(title: string) {
+      if (!title) {
```

Comment:
This logic covers the cases of passing blank strings for either variable. The test component is setup in this way, seems like a legitimate case to cover